### DISCUSSION TITLE
Panel Component: How to insert components?

### DISCUSSION THREAD

**Discussion by AlaaDotNet** [microsoft/powercat-creator-kit#264] [Q&A] [answered]
Could someone help me please. After inserting a Panel component, I set its visibility to true, in order to be able to view it. Howerver, I can't add any components inside. So how to that ? 
![image](https://user-images.githubusercontent.com/117153658/211565025-f7e1758c-5621-402b-8d05-4ceb6e91b572.png)

**Comment by yaseralhosani** [comment, marked as answer]
Hi @AlaaDotNet,

I would suggest adding any controls and components inside the Container and then set

**Container.X**: Panel_1.ContentX
**Container.Y**: Panel_1.ContentY
**Container.Width**: Panel_1.ContentWidth
**Container.Height**: Panel_1.ContentHeight
**Container.Visible**: true/false

Let me know if the problem still persists
